bertopic/_save_utils.py

```diff
@@ -88,9 +88,9 @@
 
 <details>
   <summary>Click here for an overview of all topics.</summary>
-  
+
```

Comment:
`ruff` added these new lines. I am not sure, if this effects the user. Are new lines here okay?